partner_create_by_vat/res_partner.py

```diff
@@ -191,40 +75,76 @@ def button_get_partner_data(self):
             self.write({'vat_subjected': False})
 
         if vat_number and vat_country:
-            if vat_number and vat_country and vat_country.upper()=='RO':
-                res = requests.get('http://openapi.ro/api/companies/' + str(vat_number) + '.json')
-                if res.status_code==200:
-                    res = json.loads(res.content)
-                    state = False
-                    if res['state']:
-                        state = self.env['res.country.state'].search([('name','=',res['state'].encode('utf-8').title())])
-                        if state:
-                            state = state[0].id
+            if vat_country.upper()=='RO':
+                self.write({
+                    'is_company': True,
+                    'country_id': self.env['res.country'].search([('code','=','RO')])[0].id
+                })
+                try:
+                    result = getMfinante(vat_number)
+                    name = nrc = adresa = tel = fax = zip1 = vat_s = state = False
+                    if 'Denumire platitor:' in result.keys():
+                        name = result['Denumire platitor:'].upper()
+                    if 'Adresa:' in result.keys():
+                        adresa = result['Adresa:'].title() or ''
+                    if 'Numar de inmatriculare la RegistrulComertului:' in result.keys():
```

Comment:
"Numar de inmatriculare la Registrul Comertului:" in loc de "Numar de inmatriculare la RegistrulComertului:" nu preia nrc...